test/unit/mdc-drawer/mdc-drawer.test.js

```diff
@@ -0,0 +1,228 @@
+});
+
+test('adapter#addClass adds class to drawer', () => {
+  const {component, drawer} = setupTest();
+  component.getDefaultFoundation().adapter_.addClass('test-class');
+  assert.isTrue(drawer.classList.contains('test-class'));
+});
+
+test('adapter#removeClass adds class to drawer', () => {
+  const {component, drawer} = setupTest();
+  component.getDefaultFoundation().adapter_.addClass('test-class');
+
+  component.getDefaultFoundation().adapter_.removeClass('test-class');
+  assert.isFalse(drawer.classList.contains('test-class'));
+});
+
+test('adapter#hasClass returns true when class is on drawer element', () => {
+  const {component} = setupTest();
+  component.getDefaultFoundation().adapter_.addClass('test-class');
+  const hasClass = component.getDefaultFoundation().adapter_.hasClass('test-class');
+  assert.isTrue(hasClass);
+});
+
+test('adapter#hasClass returns false when there is no class on drawer element', () => {
+  const {component} = setupTest();
+  const hasClass = component.getDefaultFoundation().adapter_.hasClass('test-class');
+  assert.isFalse(hasClass);
+});
+
+test('adapter#setStyleAppContent updates styles on the appContent', () => {
+  const {appContent, component} = setupTest(true);
+  component.getDefaultFoundation().adapter_.setStyleAppContent('display', 'inline-block');
+  assert.equal(appContent.style['display'], 'inline-block');
+});
+
+test('adapter#computeBoundingRect returns the client rect object on the root element', () => {
+  const {component} = setupTest();
+  const rect = component.getDefaultFoundation().adapter_.computeBoundingRect();
+  const rectObject = JSON.parse(JSON.stringify(rect));
+  assert.ownInclude(rectObject, {
+    bottom: 0, height: 0, width: 0, left: 0, right: 0, top: 0, x: 0, y: 0,
+  });
+});
+
+test('adapter#addClassAppContent adds class to appContent element', () => {
+  const {component, appContent} = setupTest(true);
+  component.getDefaultFoundation().adapter_.addClassAppContent('test-class');
+  assert.isTrue(appContent.classList.contains('test-class'));
+});
+
+test('adapter#addClassAppContent doesn\'t add a class if there is no appContent element', () => {
+  const {component, appContent} = setupTest();
+  component.getDefaultFoundation().adapter_.addClass('test-class');
+  assert.notExists(appContent);
+});
+
+test('adapter#removeClassAppContent removes class from appContent element', () => {
+  const {component, appContent} = setupTest(true);
+  component.getDefaultFoundation().adapter_.addClassAppContent('test-class');
+
+  component.getDefaultFoundation().adapter_.removeClassAppContent('test-class');
+  assert.isFalse(appContent.classList.contains('test-class'));
+});
+
+test('adapter#removeClassAppContent doesn\'t remove class if there is no appContent element', () => {
+  const {component, appContent} = setupTest();
+  component.getDefaultFoundation().adapter_.removeClassAppContent('test-class');
+  assert.notExists(appContent);
```

Comment:
I think we could use `doesNotThrow` here to verify that the adapter doesn't throw an error if the appContent is null/undefined. 
```
assert.doesNotThrow(component.getDefaultFoundation().adapter_.removeClassAppContent('test-class'));
```